public/styleguide/index.html

```diff
@@ -0,0 +1,51 @@
+
+<!doctype html>
+<html>
+    <head>
+        <title>Gigger Living Styleguide: </title>
+          <link rel="stylesheet" media="screen" href="/assets/application-styleguide.self-5422bf0c3e1c9487284d831795afc9dfc515604434ca60eefde2476ef11a1404.css?body=1" />
+          <link rel="stylesheet" media="screen" href="/assets/application.self-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855.css?body=1" />
+    </head>
+    <header class="header pbn" role="banner">
+        <div class="backgroundHighlight typeReversed1">
+            <div class="container">
+                <h1 class="h2 mvs">Gigger Living Styleguide: </h1>
+            </div>
+        </div>
+        <div class="backgroundLowlight typeReversed1">
+            <div class="container">
+              <span>
+                <ul class="docNav listInline">
+                    
+                        <li><a href="/styleguide/.html"></a></li>
+                    
+                </ul>
+              </span>
+            </div>
+        </div>
+    </header>
+    <div class="content">
+        <section>
+            <div class="line">
+                <div class="col cols4">
+                    <div class="componentMenu box boxBasic backgroundBasic">
+                        <div class="boxBody pan">
+                            <ul class="componentList listBorderedHover">
```

Comment:
All files under `public/styleguide` are auto generated by Hologram when it's build. You could try to correct the indentation but once you build it again, it would go back to it's previous indentation.

We don't only build once the style guide, every change it's made it have to be run again.